index.html

```diff
@@ -65,6 +65,215 @@ <h1>
+        <article class="card-work">
+            <div class="card-img">
+                <img class="img1" src="main.png" alt="image work">
+            </div>
+            <h2 class="card-title">
+                Multi-Post Stories Gain+Glory
+            </h2>
+            <ul class="box-tags">
+                <li>Rubi on rails</li>
+                <li>css</li>
+                <li>JavaScript</li>
+                <li>html</li>
+            </ul>
+            <button class="button" type="button">
+                See Project
+            </button>
+        </article>
+        <article class="card-work">
+            <div class="card-img">
+                <img class="img1" src="main.png" alt="image work">
+            </div>
+            <h2 class="card-title">
+                Multi-Post Stories Gain+Glory
+            </h2>
+            <ul class="box-tags">
+                <li>Rubi on rails</li>
+                <li>css</li>
+                <li>JavaScript</li>
+                <li>html</li>
+            </ul>
+            <button class="button" type="button">
+                See Project
+            </button>
+        </article>
+        <article class="card-work">
+            <div class="card-img">
+                <img class="img1" src="main.png" alt="image work">
+            </div>
+            <h2 class="card-title">
+                Multi-Post Stories Gain+Glory
+            </h2>
+            <ul class="box-tags">
+                <li>Rubi on rails</li>
+                <li>css</li>
+                <li>JavaScript</li>
+                <li>html</li>
+            </ul>
+            <button class="button" type="button">
+                See Project
+            </button>
+        </article>
+        <article class="card-work">
+            <div class="card-img">
+                <img class="img1" src="main.png" alt="image work">
+            </div>
+            <h2 class="card-title">
+                Multi-Post Stories Gain+Glory
+            </h2>
+            <ul class="box-tags">
+                <li>Rubi on rails</li>
+                <li>css</li>
+                <li>JavaScript</li>
+                <li>html</li>
+            </ul>
+            <button class="button" type="button">
+                See Project
+            </button>
+        </article>
+        <article class="card-work">
+            <div class="card-img">
+                <img class="img1" src="main.png" alt="image work">
+            </div>
+            <h2 class="card-title">
+                Multi-Post Stories Gain+Glory
+            </h2>
+            <ul class="box-tags">
+                <li>Rubi on rails</li>
+                <li>css</li>
+                <li>JavaScript</li>
+                <li>html</li>
+            </ul>
+            <button class="button" type="button">
+                See Project
+            </button>
+        </article>
+    </section>
+    <section class="about-me">
+        <h1>About me</h1>
```

Comment:
- [ ] The same thing for this title, It would be better to change the `h1` to `h2` or `h3` for this title.